xds/internal/balancer/clusterresolver/clusterresolver.go

```diff
@@ -175,16 +179,16 @@ func (b *clusterResolverBalancer) handleClientConnUpdate(update *ccUpdate) {
 
 // handleWatchUpdate handles a watch update from the xDS Client. Good updates
 // lead to clientConn updates being invoked on the underlying child balancer.
-func (b *clusterResolverBalancer) handleWatchUpdate(update *watchUpdate) {
+func (b *clusterResolverBalancer) handleWatchUpdate(update *resourceUpdate) {
 	if err := update.err; err != nil {
 		b.logger.Warningf("Watch error from xds-client %p: %v", b.xdsClient, err)
 		b.handleErrorFromUpdate(err, false)
 		return
 	}
 
-	b.logger.Infof("resource update: %+v", pretty.ToJSON(update.eds))
+	b.logger.Infof("resource update: %+v", pretty.ToJSON(update.p))
 	b.watchUpdateReceived = true
-	b.edsResp = update.eds
+	b.priorities = update.p
```

Comment:
General complaint:

Let's try to use more meaningful names than `p` and `chu` (above) in our structs.  1-letter things are great in small pieces of code and abbreviations are fine if they're pretty obvious and used widely (`cc = ClientConn`), but these ones will need a code search to figure out.

Reply:
both renamed